api/osrd_infra/views/sql/get_infra_no_ext.sql

```diff
@@ -0,0 +1,146 @@
+WITH infra AS (
+    SELECT %s AS id
+),
+railjson_version AS (
+    SELECT railjson_version
+    FROM public.osrd_infra_infra,
+        infra
+    WHERE public.osrd_infra_infra.id = infra.id
+),
+routes AS (
+    SELECT coalesce(json_agg(x.data), '[]'::json) AS railjson
```

Comment:
We could add the `- 'extensions'` for every objects (even if they don't have extensions yet). It keeps us from modifying the request if we modify railjson.